.vscode/settings.json.default

```diff
@@ -0,0 +1,11 @@
+{
+    "eslint.workingDirectories": [
+        "meteor",
+        "packages/blueprints-integration",
+        "packages/mos-gateway",
+        "packages/playout-gateway",
+        "packages/server-core-integration"
+    ],
+    "prettier.enable": true,
+    "sonarlint.connectedMode.project": { "projectKey": "nrkno_tv-automation-server-core" },
+}
```

Comment:
I also use `"editor.formatOnSave": true,`, should we add that too?

Reply:
I think we've agreed that that's a user preference and the default config is supposed to be minimal. Git Hooks are supposed to be the guard for code style, so I think we should leave the decision on when to format to the user.